src/vivarium_gates_iv_iron/constants/results.py

```diff
@@ -84,9 +84,7 @@
 
 POP_STATES = ("living", "dead", "tracked", "untracked")
 SEXES = ("female",)
-# TODO: revert to commented code
-# YEARS = tuple(range(2022, 2028))
-YEARS = tuple(range(2022, 2024))
+YEARS = tuple(range(2022, 2041))
```

Comment:
Should this start in 2020, since that's the start year in the config below, or are we intentionally excluding the first two years?

Reply:
It should be 2020. Good catch.